### PR TITLE
kitex: add page

### PR DESCRIPTION
**Pull request by baiyutang**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):**

### CODE REVIEW

File: pages/linux/kitex.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# kitex
```
Comment (navarroaxel) [inline pages/linux/kitex.md:1]: If the same command is available for more than 1 platform (Linux and macOS in this scenario), it should be placed on the `common/` directory.

**Review by navarroaxel — CHANGES_REQUESTED** [review]

File: pages/linux/kitex.md (hunk 1 above)
Comment (baiyutang) [reply pages/linux/kitex.md:1]: > thanks for your submission.

I know. But it is not available for  windows. So I just add page in LF & macOS.

Comment (navarroaxel) [reply pages/linux/kitex.md:1]: @baiyutang yeah, but that's the rule, if it's available for more than 1 platform, it should be placed in `common/`.

**Review by blueskyson — COMMENTED** [review]
Please move this page to `pages/common` because the usage is the same on both linux and osx.

File: pages/common/kitex.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# kitex
+
+> Kitex is a command line tool for code generation provided by the Go RPC framework Kitex.
+> At present, kitex accepts both thrift and protobuf IDLs, and supports generating a skeleton of a server side project.
+> More information: <https://www.cloudwego.io>.
```
Comment (blueskyson) [inline pages/common/kitex.md:5]:
```suggestion
> Code generation tool provided by the Go RPC framework Kitex.
> Kitex accepts both thrift and protobuf IDLs, and supports generating a skeleton of a server side project.
> More information: <https://www.cloudwego.io>.
```

File: pages.zh/common/kitex.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
+# kitex
+
+> Kitex 是 Go RPC 框架 Kitex 框架提供的用于生成代码的一个命令行工具。
+> 目前，kitex 支持 thrift 和 protobuf 的 IDL，并支持生成一个服务端项目的骨架。
+> 更多信息: <https://www.cloudwego.io>.
+
+- 生成客户端代码，项目在 `$GOPATH` 目录下：
+
+`kitex {{path/to/IDL_file.thrift}}`
```
Comment (blueskyson) [inline pages.zh/common/kitex.md:9]:
```suggestion
`kitex {{路径/到/IDL文件.thrift}}`
```

File: pages.zh/common/kitex.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+
+- 生成客户端代码，项目不在 `$GOPATH` 目录下：
+
+` kitex -module {{github.com/xx-org/xx-name}} {{path/to/IDL_file.thrift}}`
```
Comment (blueskyson) [inline pages.zh/common/kitex.md:13]:
```suggestion
` kitex -module {{github.com/xx-org/xx-name}} {{路径/到/IDL文件.thrift}}`
```

File: pages.zh/common/kitex.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+
+- 根据 protobuf IDL 文件生成客户端代码：
+
+`kitex -type protobuf {{path/to/IDL_file.proto}}`
```
Comment (blueskyson) [inline pages.zh/common/kitex.md:17]:
```suggestion
`kitex -type protobuf {{路径/到/IDL文件.proto}}`
```

**Review by blueskyson — APPROVED** [review]

**Comment by baiyutang** [comment]
@blueskyson

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**